Config.lua

```diff
@@ -307,18 +310,53 @@ local spells = {
         return not profile.general.area[info[2]].enabled
     end,
     args = {
+        selectedZone = {
```

Comment:
We have used "Area" previously, we should be consistent. Both Area and Zone correctly convey the message, we just need to pick one and stay with it.

Reply:
renamed to area